source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,99 @@
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+Engine automatically creates external network based on physical VM network with 
+physical connection to it. 
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Autodefine helps with reducing administrative tasks that needs to done with OVS 
+switch type in the cluster.
+
+## Prerequisites
+
+Cluster has to be configured with switch type OVS and valid external provider.
+The user needs to make sure that connection to external provider is valid.
+
+### Limitations
+
+* With OVS switch type in cluster only external networks are allowed to 
+  be attached to the VM interface. 
+
+* Physical network linked to any external network can't be removed. 
+
+* Existing Clusters switch cannot be changed to OVS. Currently there is no way 
+  in the Engine how to follow this change across all networks and VM interfaces.
+
+## Detailed description
+
+### Creating of new physical network
+
+When creating new network choose its name. Check the `VM network`. If vlan 
+tagging is needed, check the `Enable VLAN tagging` and put the tag into the 
+field. Please note that the newly created physical network cannot be marked as 
+external. 
+
+![Creation of physical network](../../../../images/wiki/auto-define_create.png  "New physical network")
+
+After confirmation if all conditions were met, the external network connected to 
+the physical one, will be created simultaneously. The auto-defined external 
+network is named `external_name_of_physical_network`. 
+
+![Auto-define network created](../../../../images/wiki/auto-define_created.png  "Auto-define external network")
+
+In case there are clusters with different default external providers the Engine 
+will create multiple external network for every provider and add number behind 
+this auto-defined network name. 
+
+## Implementation
+
+### Engine
+
+* Added new query `GetAllNetworksByClusterIdOvsFiltered`. The query returns 
+  only external networks for VM, that requests this query under Cluster with 
+  OVS switch type. 
+
+* Validations for all limited operations with auto-defined external network and 
+  its linked physical network. According to [Autodefine External Network - Limitations](#limitations)
+
+* Added new command `AutodefineExternalNetwork`. This command is 
+  triggered every time new network is created. If the new network is valid
+  for this operation, the Engine creates new external network. This external 
+  network is attached to corresponding clusters and linked to physical network.
```

Comment:
Could we add a brief explanation to what the command is doing?
Describe that it sends a request to all the providers on cluster, to create an external network:
{
    "network": {
        ...
        "provider:network_type": "vlan",
        "provider:physical_network": <name>,
        "provider:segmentation_id": <vlan>,
        ...
}